seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -299,16 +299,18 @@
       "id": "bad87fee1348bd9aed908626",
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
-        "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
-        "Although it is possible to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in three different ways.",
-        "Using each of the above jQuery selectors and the <code>addClass()</code> function:",
+        "Now you know three ways of targeting elements: ",
+        "by type: <code>$(\"button\")</code>, ",
+        "by class: <code>$(\".btn\")</code>, ",
+        "and by id: <code>$(\"#target1\")</code>.",
+        "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
         "Add the <code>animated</code> class to all elements with type <code>button</code>.",
         "Add the <code>shake</code> class to all the buttons with class <code>.btn</code>.",
         "Add the <code>btn-primary</code> class to the button with id <code>#target1</code>."
       ],
       "tests": [
         "assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\s*?button\\s*?(?:'|\")/gi), 'message: Use the <code>$&#40\"button\"&#41</code> selector.');",
-        "assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\s*?\\.btn\\s*?(?:'|\")/gi), 'message: Use the <code>$&#40\".btn\"&#41</code> selector.');",
+        "assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\s*?(?:button)?\\.btn\\s*?(?:'|\")/gi), 'message: Use the <code>$&#40\".btn\"&#41</code> selector.');",
```

Comment:
Why are you checking the optional existence of `button` if
1. it is already done by the assert above,
2. you need to specify only one selector per one `.addClass` method?